README.md

```diff
@@ -234,35 +238,35 @@ Instructs ESLint to try to fix as many issues as possible, see https://eslint.or
 The following steps describe the minimal steps required to successfully log in to the portal UI.
 
 1. Set up Docker, as described above: [Docker](README.md#docker)
-2. Set up your `.env` file per [Environment Variables](README.md#environment-variables)
-3. Ensure your `SSP_KK_SECRET` matches the value in the Keycloak admin console. If it does not, please try [Reset your keycloak realm](./KeycloakAdvancedSetup.md#reset-realm). If all else fails, manually generate your own secret by following: [Generating SSP_KK_SECRET](./KeycloakAdvancedSetup.md#generating-ssp_kk_secret)
-4. Run the following to install dependencies:
+1. Set up your `.env` file per [Environment Variables](README.md#environment-variables)
+1. Run the following to install dependencies:
    ```
    npm install
    ```
-5. Run the following to start the API and React front-end:
+1. Run the following to start the API and React front-end:
    ```
    npm run dev
    ```
    Successfully running this will result in the self-serve-portal opening in the browser.
-6. Run the following to build the database schema:
+1. Run the following to build the database schema:
```

Comment:
Did you mean to change the numbers to be all #1? It's fine if you did, just want to make sure it wasn't a mistake

Reply:
yeah markdown automatically converts it to the right number. I find it makes it a lot easier to maintain

Reply:
see: https://github.com/IABTechLab/uid2-self-serve-portal/tree/ajy-UID2-3354-README-improvements

Reply:
![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/137864392/6d910c60-1592-4f1c-a5e8-88fc107e2c9f)